### PR TITLE
Pybot service

### PR DESCRIPTION
**Pull request by wimo7083**
Add's pybot service url to the pods using the expected values.

Will also need to figure out how to add the dns values for staging.

### CODE REVIEW

File: dns/operationcode.org/records.tf [hunk 1]
```diff
@@ -12,9 +12,23 @@ resource "dnsimple_record" "staging_api" {
...
+  name = "pybot"
+  type = "CNAME"
+  value = "pyback-lb-197482116.us-east-2.elb.amazonaws.com"
+}
+
+resource "dnsimple_record" "staging_slack_service" {
+  domain = "${var.hosted-zone}"
+  name = "api.staging"
+  type = "CNAME"
+  value = ""
```
Comment (wimo7083) [inline dns/operationcode.org/records.tf]: I'm not sure what the permanent value is for this, waiting to hear back from Allen.

File: dns/operationcode.org/records.tf [hunk 2]
```diff
@@ -12,9 +12,23 @@ resource "dnsimple_record" "staging_api" {
   value = "${var.k8s-cluster-ingress}"
 }
 
+resource "dnsimple_record" "slack_service" {
+  domain = "${var.hosted-zone}"
+  name = "pybot"
+  type = "CNAME"
+  value = "pyback-lb-197482116.us-east-2.elb.amazonaws.com"
```
Comment (wimo7083) [inline dns/operationcode.org/records.tf]: Waiting for permanent value.

**Comment by ohaiwalt** [comment]
Could you add comments on these with the repo link, or some additional information on what they are? I also need to do that for the dashboards record. It'll help when we go to determine whether a record is necessary any longer.

Comments can be added like so:

```go
// single line comment

/*
multi
line
comment
*/
```

**Comment by ohaiwalt** [comment]
Also, question: Does sidekiq _need_ the Slack var?

**Comment by wimo7083** [comment]
What do you mean

**Comment by ohaiwalt** [comment]
> What do you mean

Is `sidekiq` sending messages to Slack? Curiosity question.

**Comment by wimo7083** [comment]
Currently sidekiq is supposed to send directly to slack api, in the near future it's going to go through our slack service (pybot) and at that point we can remove some of these slack credentials.

**Comment by ohaiwalt** [comment]
Is this ready to go? If so, I can merge after a rebase.

**Comment by wimo7083** [comment]
No, I need the correct urls it should resolve to. That's on my list to get with allen. I'm not sure how ELB and how it's setup is going to affect this.

File: dns/operationcode.org/records.tf (hunk 2 above)
Comment (wimo7083) [reply dns/operationcode.org/records.tf]: Removed this for now. Using the hardcoded values in the config.

**Comment by wimo7083** [comment]
Not exactly ready to merge. I still don't know the correct urls for both staging and prod with the pybot instance. I'll try and update today.

**Pull request merged**